graphql/resolvers.go

```diff
@@ -1658,6 +1658,15 @@ func (r *queryResolver) GithubProjectConflicts(ctx context.Context, projectID st
 	return &conflicts, nil
 }
 
+func (r *queryResolver) CanCreateProject(ctx context.Context) (bool, error) {
```

Comment:
I remember now how difficult it is to write tests to test permissions. Can we maybe instead test this through the graphql playground, either on staging or local?

Reply:
Good idea I forgot about that! Did actually catch a bug. Verified that with the role we returned true and without it we returned false.